uPortal-webapp/src/main/webapp/scripts/portal-analytics.js

```diff
@@ -16,10 +16,119 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-'use strict';
+"use strict";
 
-// Begin Portal Analytics
-window.onload = () => {
-  console.log("Window is loaded in portal analytics from scripts!!!");
-};
-// End Portal Analytics
+/**
+  * Function that captures a click on an outbound link in Analytics.
+  */
+  const outboundClick = (event) => {
```

Comment:
this appears to be top level?

```suggestion
const outboundClick = (event) => {
```

Reply:
fixed